Rohit/HTML basics remake/form.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>form</title>
+</head>
+<body>
+    <h1>Register</h1>
+ <form> 
+    <label for="First name">First name:</label> 
+    <input name="first name" type="text" placeholder="First name" required />
+    <label for="last name">Last name:</label> 
+    <input name="last name" type="text" placeholder="Last name" required /><br>
+    <br>
+    <label for="gender">gender:</label> 
```

Comment:
This doesn't require `label`. Can be direct text.

Reply:
It's okay, let it stay in `label` so that screen-readers can detect it.